core/templates/pages/topic-editor-page/subtopic-editor/subtopic-editor-tab.directive.html

```diff
@@ -0,0 +1,188 @@
+<div class="subtopic-content">
+  <div class="subtopic-content-box">
+    <div class="subtopic-left-box">
+      <md-card class="oppia-subtopics-list-card">
+        <div class="subtopic-title">
+          <p>
+            <strong style="font-size: 1.2em;">Subtopic Title</strong>
+          </p>
+          <form class="form-horizontal" role="form">
+            <input class="form-control protractor-test-subtopic-title-field"
+                   type="text"
+                   ng-blur="$ctrl.updateSubtopicTitle(editableTitle)"
+                   ng-model="$ctrl.editableTitle"
+                   maxlength="<[$ctrl.MAX_CHARS_IN_SUBTOPIC_TITLE]>"
+                   ng-change="$ctrl.resetErrorMsg()">
+            <span class="oppia-input-box-subtitle">
+              <i>
+                Subtopic title should be at most <[MAX_CHARS_IN_SUBTOPIC_TITLE]> characters.
+              </i>
+            </span>
+          </form>
+          <span class="form-text" style="font-size: smaller; color: red;" ng-if="$ctrl.errorMsg">
+            <em><[errorMsg]></em>
+          </span>
+          <span class="form-text" style="font-size: smaller; color: red;" ng-if="$ctrl.editableTitle.length === 80">
+            <em>The maximum length of subtopic title is 80 characters.</em>
+          </span>
+        </div>
+        <div class="subtopic-thumbnail">
+          <p>
+            <strong style="font-size: 1.2em;">Subtopic Thumbnail</strong>
+          </p>
+          <thumbnail-uploader filename="$ctrl.editableThumbnailFilename"
+                              update-filename="$ctrl.updateSubtopicThumbnailFilename"
+                              bg-color="$ctrl.editableThumbnailBgColor"
+                              update-bg-color="$ctrl.updateSubtopicThumbnailBgColor"
+                              allowed-bg-colors="$ctrl.allowedBgColors"
+                              aspect-ratio="'4:3'"
+                              preview-title="$ctrl.editableTitle"
+                              preview-description=""
+                              preview-description-bg-color="'#BE563C'"
+                              preview-footer="">
+          </thumbnail-uploader>
+        </div>
+        <div class="subtopic-html">
```

Comment:
Similar comment here as well. Add a title above the subtopic contents editor. Also add a 'Cancel' button to the editor to discard editing of the page contents.
![image](https://user-images.githubusercontent.com/22347970/85408491-36eb7d00-b582-11ea-98e8-cf9f020eaa51.png)